docs/api/Document.md

```diff
@@ -1,6 +1,9 @@
 # Document
 
-A document represents an item for a given model in DynamoDB. This item can created locally (meaning it's not yet saved in DynamoDB), or created from an item already stored in DynamoDB (ex. `Model.get`).
+A document (also called item in DynamoDB) represents an item for a given model in DynamoDB. This item can created locally (meaning it's not yet saved in DynamoDB), or created from an item already stored in DynamoDB (ex. `Model.get`).
```

Comment:
> (also called item in DynamoDB) represents an item

Kinda repetitive, don't you think? Read 3 more words and you know what it represents or means. Instead we add 5 more words? Just adding complexity here.

```suggestion
A document represents an item for a given model in DynamoDB. This item can created locally (meaning it's not yet saved in DynamoDB), or created from an item already stored in DynamoDB (ex. `Model.get`).
```